pkgs/development/libraries/gstreamer/vaapi/default.nix

```diff
@@ -5,7 +5,7 @@
 
 stdenv.mkDerivation rec {
   name = "gst-vaapi-${version}";
-  version = "1.14.0";
+  version = "1.14.2";
```

Comment:
The hash also needs an update.

Reply:
Thanks for finding this mistake. Solved :)

Reply:
I just tried to build `gst_all_1`... all the rest had succeeded.

Reply:
But now it works?